public/app/plugins/datasource/postgres/postgres_query.ts

```diff
@@ -0,0 +1,285 @@
+import _ from 'lodash';
```

Comment:
This class should also be fairly easy to extract to common base class/component eventually :+1: